Xamarin.Forms.Core/ImageSource.cs

```diff
@@ -68,7 +67,19 @@ public static ImageSource FromResource(string resource, Type resolvingType)
 
 		public static ImageSource FromResource(string resource, Assembly sourceAssembly = null)
 		{
-			sourceAssembly = sourceAssembly ?? Assembly.GetCallingAssembly();
+			if (sourceAssembly == null)
+			{
+				MethodInfo callingAssemblyMethod = typeof(Assembly).GetTypeInfo().GetDeclaredMethod("GetCallingAssembly");
+				if (callingAssemblyMethod != null)
+				{
+					sourceAssembly = (Assembly)callingAssemblyMethod.Invoke(null, new object[0]);
```

Comment:
This reflection will be a problem when use .netcore runtime. (see https://bugzilla.xamarin.com/show_bug.cgi?id=60731).

Cause:
In .NET standard, the Assembly.GetCallingAssembly() has a [System.Security.DynamicSecurityMethod] attribute set (https://github.com/dotnet/coreclr/blob/13e7c4368da664a8b50228b1a5ef01a660fbb2dd/src/mscorlib/src/System/Reflection/Assembly.CoreCLR.cs#L248), which means callingAssemblyMethod.Attributes contains MethodAttributes.RequireSecObject flag, which in turn means that it cannot be invoked: (https://github.com/dotnet/coreclr/blob/97c58ac4fce27b7796206a59eea0ca27cb49fe1a/src/mscorlib/src/System/Reflection/RuntimeMethodInfo.cs#L48).

Reply:
do you think this will work? netstandard2.0 should be the prefer way

Reply:
yes it looks good :) thank you.